lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -194,6 +192,17 @@ def post(self, request, *args, **kwargs):
         self._handle_marketing_opt_in(request, course_key, user)
         return response
 
+    def _pass_along_user_cookie_to_server(self, server_session, request, cookie_name):
```

Comment:
I think there is a more accurate name for this method.  Maybe, `_add_request_cookie_to_api_session`